CME/dataobjects/CMEAccount.php

```diff
@@ -129,20 +129,34 @@ public function getEarnedCMECreditHours()
 	}
 
 	// }}}
-	// {{{ public function getEarnedCMECreditHoursByProvider()
+	// {{{ public function getEarnedCMECreditsByProvider()
 
-	public function getEarnedCMECreditHoursByProvider(CMEProvider $provider)
+	public function getEarnedCMECreditsByProvider(CMEProvider $provider)
 	{
-		$hours = 0;
+		$class_name = SwatDBClassMap::get('CMECreditWrapper');
```

Comment:
This class isn't required in the file, throwing an exception. Fixed with https://github.com/nburka/Cme/pull/4